Triphouse-project/style.css

```diff
@@ -0,0 +1,80 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
+/* content */
+body {
+    font-family: 'Roboto', sans-serif;
+}
+.content {
+    margin: 0px auto;
+    text-align: center;
+    max-width: 1440px;
+    height: 569px;
+}
+.content h2 {
+    font-size: 40px;
+    font-weight: 500;
+    text-align: center;
+    margin-top: 120px;
+}
+.content .content_items {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.content h2::after {
+    content: '';
+    position: relative;
+    display: block;
+    width: 100px;
+    height: 4px;
+    margin: 0px auto;
+    top: 20px;
+    left: 0;
+    right: 0;
+    background: #f5bd41;
+}
+.content_item img {
+    display: flex;
```

Comment:
Why is it flex ?
Add style how on image
![2021-04-12_15h50_14](https://user-images.githubusercontent.com/19935417/114397082-f5769d80-9ba6-11eb-98ab-69a90ee7c6d3.png)

Reply:
fixed the error